assets/component-menu-drawer.css

```diff
@@ -223,6 +223,10 @@ details[open].menu-opening > .menu-drawer__submenu {
   margin-right: 1rem;
 }
 
+.menu-drawer__account:hover .icon-account {
+  transform: scale(1.07);
+}
```

Comment:
I think it is since we are able to resize it and it's not on mobile only. This way we ensure there is a hover state 🤔 

https://screenshot.click/10-15-sh69x-vyz3n.png